helm/app/values.yaml

```diff
@@ -37,12 +37,6 @@ ingress:
       hosts:
         - go-api-boilerplate.local
 
-kubernetes-dashboard:
```

Comment:
Lets not create new release. Lets try to keep everything in the same one. Move newly crated dashboard release in the app one.